src/test/regress/expected/non_super_user_object_metadata.out

```diff
@@ -0,0 +1,303 @@
+SET citus.enable_ddl_propagation TO OFF;
+CREATE SCHEMA local_schema;
+SET search_path TO local_schema;
+-- Create type and function that depends on it
+CREATE TYPE test_type AS (f1 int, f2 text);
+CREATE FUNCTION test_function(int) RETURNS test_type
+    AS $$ SELECT $1, CAST($1 AS text) || ' is text' $$
+    LANGUAGE SQL;
+-- Create various objects
+CREATE TYPE mood AS ENUM ('sad', 'ok', 'happy');
+CREATE SEQUENCE test_sequence;
```

Comment:
to make this a little more interesting, should we create one more schema, and create the test_sequence here?

Two more test ideas that'd make me more comfortable:
1) create an extension on a completely separate schema, and make sure that the schema is created on the workers:
```SQL
CREATE SCHEMA extension_schema;
CREATE EXTENSION ltree WITH SCHEMA extension_schema;
-- similar queries on pg_extension
```

2) ALTER FUNCTION a_distributed_function SET SCHEMA a_new_schema;
    ALTER TYPE a_distributed_type SET SCHEMA a_new_schema_2;
and, make sure that a_new_schema and a_new_schema_2 are distributed / in workers' pg_dist_object as well.